manifest/manifest.go

```diff
@@ -148,6 +194,47 @@ func (m *Manifest) Equal(o *Manifest) bool {
 
 }
 
+func (m *Manifest) Validate() error {
+	switch {
+	case m == nil:
+		return fmt.Errorf("invalid manifest: manifest is nil!")
+	case m.NetworkName == "":
+		return fmt.Errorf("invalid manifest: network name must not be empty")
+	case m.BootstrapEpoch < m.EC.Finality:
```

Comment:
```suggestion
	case m.BootstrapEpoch <= m.EC.Finality:
```

Reply:
This _should_ work as-is, and I don't think it's important enough to change.